modules/ping-protection/models/LeaverData.js

```diff
@@ -0,0 +1,26 @@
+const { DataTypes, Model } = require('sequelize');
+
+module.exports = class LeaverData extends Model {
```

Comment:
```suggestion
module.exports = class PingProtectionLeaverData extends Model {
```
ensure module name is contained in class name to avoid sequelize issues, also name is not OOP

Reply:
Please fix for all other models as well